test/integration/targets/mail/files/smtpserver.py

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2018, Dag Wieers (@dagwieers) <dag@wieers.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+import asyncore
+import os.path
+import ssl
+import sys
+
+# Handle TLS and non-TLS support
+try:
+    import smtpd_tls
+    HAS_TLS = True
+except ImportError:
+    import smtpd
+    HAS_TLS = False
+
+# Handle custom ports
+port = '25:465'
+if len(sys.argv) > 1:
+    port = sys.argv[1]
+ports = port.split(':')
+if len(ports) > 1:
+    port1, port2 = int(ports[0]), int(ports[1])
+else:
+    port1, port2 = int(port), None
+
+if HAS_TLS:
+    # Handle custom certificate
+    basename = os.path.splitext(sys.argv[0])[0]
+    certfile = basename + '.crt'
+    if len(sys.argv) > 2:
+        certfile = sys.argv[2]
+
+    # Handle custom key
+    keyfile = basename + '.key'
+    if len(sys.argv) > 3:
+        keyfile = sys.argv[3]
+
+    print('Using %s and %s' % (certfile, keyfile))
+    ssl_ctx = ssl.create_default_context(ssl.Purpose.CLIENT_AUTH)
```

Comment:
The test server is failing to start with the error:

`AttributeError: 'module' object has no attribute 'create_default_context'`

On Ubuntu 14.04 we only have Python 2.7.6, but `create_default_context` was added in Python 2.7.9:

https://docs.python.org/2/library/ssl.html#ssl.create_default_context